modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,32 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Spell Checker Pro 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.3.0 includes three additions:
+
+==== New `mceSpellcheckUpdate` command
+
+The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words.
+
+This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.
+
+==== New `SpellcheckerUpdated` event
+
+The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display any new misspelt words, once suggestions are provided for those misspelt words.
+
+This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog.
+
+NOTE: This event is currently not supported in As-you-type mode.
+
+==== New `getSpellingErrors()` API
+The new `getSpellingErrors()` API allows users to retrieve a list of each misspelling in the editor content.
+
+[NOTE]
+Read more about xref:ROOT:introduction-to-tiny-spellchecker.adoc[new Spell Checker Pro features here].
```

Comment:
```suggestion
=== Spell Checker Pro 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

**Spell Checker Pro** 3.3.0 includes three additions:

==== New `mceSpellcheckUpdate` command

The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words.

This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.

==== New `SpellcheckerUpdated` event

The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display new misspelt words once suggestions are provided for those misspelt words.

This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog.

NOTE: This event is not currently supported in As-you-type mode.

==== New `getSpellingErrors()` API

The new `getSpellingErrors()` API allows users to retrieve a list of each misspelling in the editor content.

For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Introduction to Spell Checker Pro].
```

1. Copy-edits.
2. Turned NOTE admonition into the standard _for information_ line.